packages/assert/src/assert.js

```diff
@@ -56,27 +64,28 @@ function fail(optDetails = details`Assert failed`) {
 
 // assert that expr is truthy, with an optional details to describe
 // the assertion. It is a tagged template literal like
-// ```assert(expr, details`....`);```
+// ```js
+// assert(expr, details`....`);`
+// ```
 // If expr is falsy, then the template contents are reported to the
 // console and also in a thrown error.
 //
 // The literal portions of the template are assumed non-sensitive, as
 // are the `typeof` types of the substitution values. These are
-// assembles into the thrown error message. The actual contents of the
+// assembled into the thrown error message. The actual contents of the
 // substitution values are assumed sensitive, to be revealed to the
 // console only. We assume only the virtual platform's owner can read
 // what is written to the console, where the owner is in a privileged
 // position over computation running on that platform.
 //
-// The optional `details` can be a string for backwards compatibility
-// with nodejs assertion library.
+// The optional `optDetails` can be a string for backwards compatibility
+// with the nodejs assertion library.
 function assert(flag, optDetails = details`check failed`) {
   if (!flag) {
     console.log(`FAILED ASSERTION ${flag}`);
     fail(optDetails);
   }
 }
-assert.details = details;
```

Comment:
the advantage of `assert.details` is that it's easier to pass the single `assert` as an endowment, and it does match a certain style of usage that I've seen with various libraries.  I understand not wanting to support more than one pattern, but the first argument might be compelling.

Reply:
Sure, I could see one or the other. But not both. Of the two, I prefer what we kept.